### PR TITLE
Add wifi driver

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by nikhil-zinjurde-imgtec** [comment]
test with jenkins - dev

File: uccp420wlan/files/etc/init.d/uccp420wlan [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/bin/sh /etc/rc.common
+
+. /lib/functions/mac.sh
+
+START=10
+STOP=10
+
+start() {
+	generate_mac $WIFI_OTP_REG0
```
Comment (abhijit-mahajani-imgtec) [inline uccp420wlan/files/etc/init.d/uccp420wlan]: generate_mac uses proddata functions so you need to added DEPENDS for proddata too.
However we want to move away from proddata. so can you use wifiaddr from uboot env?
Just setenv wifiaddr for the time being and check if it works?

**Review by abhijit-mahajani-imgtec — CHANGES_REQUESTED** [review]

**Comment by nikhil-zinjurde-imgtec** [comment]
retest this please

**Comment by nikhil-zinjurde-imgtec** [comment]
retest this please

**Comment by nikhil-zinjurde-imgtec** [comment]
retest this please

**Comment by nikhil-zinjurde-imgtec** [comment]
retest this please

**Comment by nikhil-zinjurde-imgtec** [comment]
retest this please

**Comment by nikhil-zinjurde-imgtec** [comment]
retest this please

**Comment by nikhil-zinjurde-imgtec** [comment]
retest this please

**Comment by nikhil-zinjurde-imgtec** [comment]
retest this please

**Comment by abhijit-mahajani-imgtec** [comment]
please update readme too

**Comment by nikhil-zinjurde-imgtec** [comment]
@francois-berder-imgtec please add PR directly on master branch, we will soon be closing off the dev branch. So, please close this PR and create a new one on master

**Comment by abhijit-mahajani-imgtec** [comment]
Closing this one since its done for master branch in #102

**Pull request closed without merging**